guava/pom.xml

```diff
@@ -57,6 +57,10 @@ com.google.common.*;version="${version.guava.osgi}",
         <groupId>org.moditect</groupId>
         <artifactId>moditect-maven-plugin</artifactId>
       </plugin>
+      <plugin>
```

Comment:
Not sure why this would be needed: parent pom (`jackson-datatypes-collections`) has it with exclusion for `failing` (which is to allow running failing-for-now tests from IDE without blocking build).

Reply:
The `mvn clean install` output with and without this addition is different. Apparently, Guava's tests are not run without it. The spec in `jackson-datatypes-collections` is in `<pluginManagement>` element, while here it's in `<build><plugins>`.

Reply:
That is odd: for me `mvn clean install` from main level does run all the tests on `master`.
Note, too, that none of child projects has separate surefire run settings and tests are run.

Reply:
You are right. maven-surefire-plugin config in `<build><plugins>` is actually inherited from Jackson's `oss-parent` pom. This could be verified by generating Effective Poms. And now, I cannot reproduce the tests not running without these configs (they run now for me), maybe it was my mental glitch.

Reply:
Ok. Stuff happens, I just wanted to make sure I understood reasoning -- I would not want tests to be accidentally skipped for sure.

I hope to merge this soon, today or tomorrow. Thank you for all the work here!